ServiceAreaParkingStatus/Presentation/Main/ViewController.swift

```diff
@@ -179,7 +179,7 @@ extension ViewController: ParkingManagerDelegate {
     }
     
     func didUpdateParking(_ parkingManager: ParkingManager, parking: ParkingData) {
-        self.serviceAreaArray = parking.data.map{ String($0.휴게소명) }
+        self.serviceAreaArray = parking.data.map{ $0.serviceArea }
```

Comment:
각자의 스타일이 있지만, 일반적으로 클로저에 사용되는 괄호는 앞뒤를 띄워 주는 것이 일반적입니다. (`SwiftLint` 에서 사용하는 [Kodeco's Swift Style Guide](https://github.com/kodecocodes/swift-style-guide) 기준)
```suggestion
        self.serviceAreaArray = parking.data.map { $0.serviceArea }
```

Reply:
앗 그렇군요, 알려주셔서 감사합니다.
코드 컨벤션 기준을 좀 더 알고 싶었는데 `SwiftLint`를 설치해서 습관을 들여야겠습니다.